src/gettingstarted/template/cli-install.md

```diff
@@ -0,0 +1,58 @@
+# Start with a template
+
+## Next Steps: Installing the Platform.sh CLI
+
+With all of the requirements met, install the CLI to start developing with Platform.sh.
+
+<asciinema-player src="/videos/asciinema/verify-cli-extended.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Install the CLI**
+
+    In your terminal run the following command:
+
+    * **Installing on OSX or Linux**
+
+       ```bash
+       curl -sS https://platform.sh/cli/installer | php
+       ```
+
+    * **Installing on Windows**
+
+       ```bash
+       curl https://platform.sh/cli/installer -o cli-installer.php
+       php cli-installer.php
+       ```
+
+2. **Authenticate and Verify**
+
+   Once the installation has completed, you can run the CLI in your terminal with the command
+
+   ```bash
+   platform
+   ```
+
+   You should now be able to see a list of your Platform.sh projects, including the template you made in this guide. You can copy its *project ID* hash, and then download a local copy of the repository with the command
+
+   ```bash
+   platform get <project ID>
+   ```
+
+   With a local copy, you can create branches, commit to them, and push your changes to Platform.sh right away!
+
+   ```bash
+   git push platform master
```

Comment:
The `platform master` part is unnecessary, since there is an upstream tracking branch set.